programmers/20210504/두-정수-사이의-합.js

```diff
@@ -0,0 +1,8 @@
+function solution(a, b) {
+    if (Math.abs(a - b) % 2 == 0) {
+        return ((a + b) * (Math.abs(a - b) + 1)) / 2
+    }
+    else {
+        return (a + b) * ((Math.abs(a - b) / 2) + 0.5)
+    }
```

Comment:
if문에서 return을 했기 떄문에 else문을 안쓰셔도 됩니다.
```suggestion
function solution(a, b) {
    if (Math.abs(a - b) % 2 == 0) {
        return ((a + b) * (Math.abs(a - b) + 1)) / 2;
    }
    return (a + b) * ((Math.abs(a - b) / 2) + 0.5);
```